test/integration/IsNotNull/is not null.js

```diff
@@ -0,0 +1 @@
+typeof aaa !== "undefined"
```

Comment:
This should be

```javascript
-
typeof a !== "undefined"
-
```

Note the dashes on the first and last lines. These are so commands that should actually end up in 0, 1, or 2 blank lines can be distinguished during tests.